src/main/java/mira/Storage.java

```diff
@@ -89,14 +89,14 @@ private Task createTaskFromFile(String[] parts) throws MiraException {
         String description = parts[2];
```

Comment:
Perhaps you could use a more descriptive name for the split description of the task?